src/java/boa/datagen/DefaultProperties.java

```diff
@@ -74,6 +74,11 @@
 
 	public static String localDataPath = null;
 	
+	// for ML
+	public static String localOutput = null;
+	public static String HADOOP_SEQ_FILE_LOCATION = "hdfs://master-link-1/outputModel";
```

Comment:
this needs to be configurable somehow... this isn't even the right path.

look here for an example:
https://github.com/boalang/compiler/blob/2f03515bc7f0cebe991d17a16a56b83e5db54ef2/src/java/boa/functions/BoaAstIntrinsics.java#L362

Reply:
I have removed HADOOP_SEQ_FILE_LOCATION and fixed the path in MLAggregator.java